### DISCUSSION TITLE
What are JWST Little Red Dots?

### DISCUSSION THREAD

**Discussion by LifeIsStrange** [a-cosmology-group/acg#248] [A Cosmology Group]
Little red dots can be abundantly observed on the background of deep JWST pictures.
They are seemingly too small to allow either a photometric nor spectroscopic redshift estimate but they are likely to be high redshift objects.
https://arxiv.org/abs/2407.05094
https://arxiv.org/abs/2406.10341
https://arxiv.org/abs/2407.04777
https://arxiv.org/abs/2406.18207
https://arxiv.org/abs/2406.10329
https://arxiv.org/abs/2404.19010

> If one assumes LRDs' rest-optical light is dominated by stars, the star-to-dust ratio is a factor of ∼100× larger than expected from dust formation models;

> remarkable abundance of LRDs implies a dominant mode of early galaxy/SMBH growth.

**Comment by mikehelland** [comment]
I think it's a bit early to say for sure, but probably just bright, active galaxies.

The Hubble eXtreme deep field images took 2 million seconds, or 23 days:

https://en.wikipedia.org/wiki/Hubble_Ultra-Deep_Field#Hubble_eXtreme_Deep_Field

JWST's deep field image was taken in 20 hours:

https://bigthink.com/starts-with-a-bang/jwst-surpasses-hubbles-deepest-image/

I suspect if JWST stared at one point in the sky for 23 days, it'd basically be all red.

**Reply by LifeIsStrange** [reply]
I concur about the extreme potential of newer or larger JWST deep fields.
Literally 4 of the five highest confirmed redshift galaxies are from JADES, so 4 record breaking from a single 20h observation...
(of course it takes more times for confirmations (albeit not much contrary to popular belief, JWST is simply very badly managed in terms of time allocations))

**Reply by mikehelland** [reply]
>JWST is simply very badly managed in terms of time allocations

How so?

**Reply by RedshiftDrift** [reply]
JWST time is strongly biased toward discovering and analysing exoplanets, trying to find one with an atmosphere that could support life.  The public is more interested in 'aliens' than in finding distant galaxies.
It's a publicly funded project, they have to go with the will of the masses!

**Reply by LifeIsStrange** [reply]
While I believe JWST observation time is badly allocated I don't believe they allocate a large chunk (<10%?) to exoplanet observation.
I might be wrong, I actually don't know, given that my arxiv feed is currently only made of
> astro-ph.CO - Cosmology and Nongalactic Astrophysics
And
> astro-ph.GA - Astrophysics of Galaxies
I don't browse much
> astro-ph.EP - Earth and Planetary Astrophysics

While I have this observation bias, my reasoning is as follow:
Exoplanet research is quite interesting, in fact it is my desire to find alien life or habitable planets that drove me to cosmology initially.
But the idea that it is Hubble or JWST observations that allowed exoplanets discoveries is a misconception I had.

![image](https://github.com/user-attachments/assets/98be0b08-5140-4cfc-9b71-5b5f92a1875b)

Those are the new exoplanets discoveries over time, it is apparent that the transit method pursued by Kepler, TESS, Cheops and PLATO considerably outperforms all other methods combined, by far.
(transit in green, direct detections (jwst, hubble in blue) and radial velocities (Gaia, IFU spectrographs) in red).

Something that is striking, and that I never see discussed, is that Kepler successor, TESS seems to be a gigantic failure and the reason isn't clear to me.
It detects ~500% less exoplanets than Kepler (as can be seen post 2016), despite having a "much wider" field of view. 
So the question being:
Is this because TESS has chosen an inferior tradeoff? (wider field of view but lower resolution/sensitivity?) 
Is this because Kepler has exhausted all the low hanging fruits in the Milky Way? (unlikely given fov?) 
Are the exoplanets comparable? The ratio of short versus long periods orbiting planets, the ratio of giant versus "small" planets, etc 

This debate is very important because soon, the revolutionary PLATO will be launched and while their main paper, shows ambitious discovery rates and asteroseismology constraints, I am skeptical of the approach of using multiple small mirrors. I don't believe they will combine them for aperture synthesis? If so then we will have high parallelism but significantly lower sensitivity and resolution to faint planets than Kepler or a super Kepler project... 

In parallel I expect the amount of exoplanets discovered through radial velocities and microlensing to boom in the next few years albeit it should remain below the transit method.

Exoplanets direct imaging is incredible as shows this interpolation
https://en.m.wikipedia.org/wiki/List_of_directly_imaged_exoplanets#/media/File%3AHR_8799_Orbiting_Exoplanets.gif
But JWST has directly imaged none? Or a negligible number
https://en.m.wikipedia.org/wiki/List_of_directly_imaged_exoplanets

I have read it has been used spectroscopically to constrain some exoplanets atmospheres which is interesting but I don't believe this has been done enough times to warrant much observation time.

Edit the Roman telescope is "expected" to detect 2500 planets ~half the rate or Kepler, which is pretty impressive for an optical telescope, if it scales to the amount of pixels then Xuntian could be expected to detect 25000 planets!
If it scales mostly via spectrography Xuntian will have only a couple more than Roman. Of courses JWST doesn't count as it has a ridiculously smaller field of view.
Overall for the transit method I don't know wether the PLATO or ARIEL approach is better.

**Reply by mikehelland** [reply]
You can search for jwst on arxiv like this:

https://arxiv.org/search/?query=jwst&searchtype=all&source=header

At least half the papers are exoplanet related.

As far as directly imaging an exoplanet, I don't think you'd see much. This is NIRCam's view of Titan:

![image](https://github.com/user-attachments/assets/8336ccdb-a91f-418c-a298-8d91d5e2ce44)

Most of the exoplanet stuff uses NIRSpec to analyze the atmospheres of other worlds.

**Reply by LifeIsStrange** [reply]
Thanks Mike, yes JWST is underpowered for direct imaging except maybe for giant planets, vs LUVOIR .
Even for spectroscopy it seems underpowered Cf the atmospheric signal to noise ratio in the only exoplanet it confirmed.

https://arxiv.org/search/advanced?advanced=&terms-0-operator=AND&terms-0-term=JWST&terms-0-field=all&terms-1-operator=AND&terms-1-term=astro-ph.EP&terms-1-field=all&classification-physics=y&classification-physics_archives=astro-ph&classification-include_cross_list=include&date-filter_by=all_dates&date-year=&date-from_date=&date-to_date=&date-date_type=submitted_date&abstracts=show&size=50&order=-announced_date_first

The percentage of exoplanet papers *mentioning* JWST is I find extremely higher than I expected:
940/3035, meaning that 31% of papers mentioning JWST are exoplanet research papers.
However my conjecture is that the field of exoplanet research is simply instrument starved and must therefore spam "JWST" mentions independently of the actual number of observations made by JWST on exoplanets. 
Given there is only a single exoplanet confirmation made by JWST I don't think it has been much used.

**Reply by mikehelland** [reply]
There are better instruments for detecting exoplanets. There aren't better instruments for examining them.

**Reply by LifeIsStrange** [reply]
So I have skimmed at the abstract of all papers in exoplanets research mentioning JWST from now to march, in order to get a better idea.
I can confirm that JWST does publish new observations of exoplanets at a mild frequency (a handful per month).
Among those most are either observations of m dwarf stars (hence an indirect information separately used for host planets), of brown or ultracool dwarfs, or of jupiter class planets AKA giants and supergiants. A few are observations of dust clouds, discs or asteroids, that are indirectly related to the topic of planetary formation.

There seems to be a few superstar JWST observations that are mentioned much more often than others, especially WASP-39 b.

A great paper on the topic is
https://arxiv.org/abs/2404.15430

I don't have statistics for the number of total JWST observations of exoplanets but figure 6 give an upper bound:
There are rhoughly 200 exoplanets that have "high quality" spectroscopic observations in total, the percentage of JWST being unknown as simultaneously to the launch of JWST we have seen an explosion in the number of wide field spectrographs such as e.g. EXPRESSO, APOGEE, etc.
Of them, there are only 9 planets (5%) that have detected carbon AND oxygen.
Overall I don't *believe* more than 10-20% of JWST time is being cannibalized by exoplanet science.

Edit the database is on github
https://www.arjunsavel.com/hires_papers
https://x.com/ArjunSavel/status/1582563317515632640

**Reply by mikehelland** [reply]
It's the second largest usage (this is from year 1):

https://www.stsci.edu/files/live/sites/www/files/home/jwst/science-planning/user-committees/jwst-users-committee/_documents/jstuc-0421-jwst-cycle1-review-package.pdf#page=20

But I'm not claiming exoplanets are "cannibalizing" JWST time. It's just a fact that it spends a lot of time on that. You had claimed it is poorly managed. I just asked how.

**Reply by LifeIsStrange** [reply]
Thank you this is exactly what I was looking for.
There seems to be roughly 50% of JWST time allocated for exoplanetary science on those proposals.

Note that the document seems to not be the final schedule but one might guess it is representative of the final schedule.
I see one discrepancy though,
The paper states:
Instruments (time): MIRI 28.1%, NIRCam 24.4%, NIRISS 6.7%, NIRSPEC 40.8%
 • Imaging 30% vs Spectroscopy 70%

But I had read a recent paper saying that MIRI is considerably less used than NIRSPEC, I don't remember the exact number but an over 10X difference IIRC.
Anyway I find it quite amazing that with ~50% of JWST time they have measured the spectra of less than 50, already confirmed, planets, one has to wonder what they actually do with this time.

As a reminder there is this tool
https://webbtelescope.org/science/the-observatory/what-is-webb-observing-now
Here are all proposals descriptions
https://www.stsci.edu/jwst/science-execution/approved-programs/general-observers/cycle-1-go

I have high hopes for those proposals
https://www.stsci.edu/jwst/science-execution/program-information?id=5324
https://www.stsci.edu/jwst/science-execution/program-information?id=4598

And the highest value one
https://www.stsci.edu/jwst/science-execution/program-information?id=5224

**Reply by LifeIsStrange** [reply]
So I have read a lot about exoplanet research and there are many discoveries I will write about later but to start:
I did NOT expect such performance from GAIA.
1) I wasn't familiar with the difference between absolute versus relative astrometry (Gaia is the former while Hubble FGS and VLT GRAVITY interferometers are the latter)
2) I dismissed the astrometric method for planet detection based on the fact that historically it yielded a negligible amount of discoveries and because the current big DR3 data release only yielded ~80 candidates.
GAIA by DR5 is expected to discover around 10000 exoplanets which is 3 times the discovery rate of Kepler!
This extreme non linearity is possible because GAIA new data releases improve global astrometric constraints
> precisions of 30, 10, and 7 µas for the third, fourth, and fifth Gaia data releases (DR3, DR4, and DR5) 

Such non linearity in accuracy should also be expected for gravity theories. 
One has to wonder if we kept GAIA alive for a few more years if such non linearity would continue and yield over a million of explonanets discoveries. 
If so then GAIA will outperform any other exoplanet discovery methods. 
Unfortunately at least until a DR6, this should be limited to planets of a size >= to saturn.
https://arxiv.org/abs/2403.08226

Btw figure 1) below is great to see the selection biases of each methods (astrometry not presented)
https://arxiv.org/abs/2404.15430

The Ariel telescope is also very interesting, Kepler and platos will enable a massive amount of new exoplanets discoveries and contraints on the host star, planet orbit, radius and a few others parameters, but there is a gap in atmospheric science that require mass high performance spectrometry, which is what JWST currently provide but only for a few dozens of planets. Ariel, like possibly Roman/xuntian will fill this gap, but most importantly Ariel comes with 3 photometers while Kepler had only one so it might allow major advances
https://link.springer.com/article/10.1007/s10686-021-09777-x

**Reply by LifeIsStrange** [reply]
So I am currently reading about the topic of exoplanets direct imaging (AKA High contrast imaging)
You can see all direct imaged (giant) planets here
https://en.m.wikipedia.org/wiki/List_of_directly_imaged_exoplanets
As shown in my previous comment figure 1, while viewing planet orbits is amazing it is currently limited to planets that have the highest mass and largest orbits. 
However this might be changing! 
With better instruments this is expected to change, forecasts of the incoming SPHERE+ upgrade aims to target jupiter mass planets (300 earths) and with orbits lengths near earth length.
Cf figure 8 below 
https://arxiv.org/abs/2003.05714
One might wonder how much that will scale with future instruments, 
So I have read about High contrast planet imagers and remarkably, while it is one of the first goals of astronomy that comes to mind, it is an extremely underfunded field... 
It's simple almost all pictures of planets come from a single and old instrument: SPHERE at the 8meter VLT telescope. And it's performance has been quite impressive for a first generation instrument. For some reasons not a single other ground based telescope has imaged other planets (except maybe until very recently). 
I've found around 3 other large telescopes that aims to develop a SPHERE competitor. 
In addition, the E-ELT and GMT next Gen telescopes will have high contrast cameras like SPHERE.
And this is the most striking, mind blowing thing, from what I read from the SPHERE+ upgrade, improving the performance of planet imaging is largely atmosphere limited and adaptative optics techniques have mostly reached their limits, meaning the E-ELT is not guaranteed to do much better. 
It is absolutely crazy that we haven't yet nor aren't even planning to send a high contrast camera to space, since the medium IR band is optimal, herschel or JWST would have enabled revolutionary planet imaging but their cameras aren't made for it. Contrary to my previous belief, while mirror size matters, it is not the main Bottleneck here. 
Future exoplanet space telescopes like PLATO and Ariel have zero imaging capability.
The fact we don't do this huge advance is even more sad given that there is a synergy between astrometric data and direct imaging, this has recently allowed to find a jupiter mass planet aka it lowers the minimum mass detection.
https://arxiv.org/abs/2302.06213

This is why I really want to read this paper about the future of imaging, sadly I can't find the pdf online, if someone could send it to me that would be great. 
https://www.researchgate.net/publication/380524755_Future_Exoplanet_Research_High-Contrast_Imaging_Techniques

Here is an older review
https://www.prophy.science/article/2702618-Observing-Exoplanets-with-High-Dispersion-Coronagraphy-I-The-scientific-potential-of-current-and-next-generation-large-ground-and-space-telescopes/
And 
https://arxiv.org/abs/1807.07100
This paper is extremely interesting, it shows that optimal planet detection is currently optimal (in space) with low resolution spectrographs (R=400) though future noise reductions techniques might make higher resolution spectrographs better.
Not only that but it says that a 12 meter (LUVOIR) or even 4 meter space telescope (habex) can detect nearby earth sized planets. (now how much nearby I'm not sure) 


ELT instrument:
https://www.prophy.science/article/115097805-METIS-high-contrast-imaging-design-and-expected-performance/
https://arxiv.org/abs/2407.10588
https://www.prophy.science/article/43103610-The-MICADO-first-light-imager-for-the-ELT-overview-operation-simulation/
https://www.prophy.science/article/129540751-PCS-A-Roadmap-for-Exoearth-Imaging-with-the-ELT/
https://www.prophy.science/article/139950333-Direct-imaging-and-spectroscopy-of-exoplanets-with-the-ELT-HARMONI-high-contrast-module/
https://www.prophy.science/article/158324290-HARMONI-at-ELT-system-analysis-and-performance-estimation-of-the-high-contrast-module/
https://www.prophy.science/article/141938973-The-ExoLife-Finder-ELF-Telescope-new-adaptive-optics-and-hybrid-dynamic-live-optical-surfaces-strategies/

GMT:
https://www.prophy.science/article/157261588-Phasing-the-Giant-Magellan-Telescope-with-the-holographic-dispersed-fringe-sensor/
https://www.prophy.science/article/172725510-GMagAO-X-A-First-Light-Coronagraphic-Adaptive-Optics-System-for-the-GMT/

Gemini telescope:
https://arxiv.org/abs/2404.15528
https://www.prophy.science/article/159840485-Deployment-of-focal-plane-WFS-technologies-on-8-m-telescopes-from-the-Subaru-SPIDERS-pathfinder-to-the-facility-class-GPI-2-0-CAL2-system/

Keck telescope:
https://www.prophy.science/article/179926845/
https://www.prophy.science/article/159589533-Fiber-fed-high-resolution-infrared-spectroscopy-at-the-diffraction-limit-with-Keck-HISPEC-and-TMT-MODHIS-status-update/

Subaru https://www.prophy.science/article/143537705-First-on-sky-demonstration-of-spatial-Linear-Dark-Field-Control-with-the-vector-Apodizing-Phase-Plate-at-Subaru-SCExAO/
MKID Exoplanet Camera (MEC; Walter et al. 2020)
https://www.prophy.science/article/159872745-The-JWST-Early-Release-Science-Program-for-Direct-Observations-of-Exoplanetary-Systems-I-High-Contrast-Imaging-of-the-Exoplanet-HIP-65426-b-from-2-16-mu-m/

About limitations
https://iopscience.iop.org/article/10.1088/0004-637X/792/2/97#skip-to-content-link-target

Space
https://www.prophy.science/article/630890-High-contrast-Imaging-from-Space-Speckle-Nulling-in-a-Low-Aberration-Regime/
https://www.prophy.science/article/845365-Wavefront-error-correction-and-Earth-like-planet-detection-by-a-self-coherent-camera-in-space/
https://www.prophy.science/article/1164929-Maximizing-the-ExoEarth-Candidate-Yield-from-a-Future-Direct-Imaging-Mission/
https://www.prophy.science/article/1253456-Lower-Limits-on-Aperture-Size-for-an-ExoEarth-Detecting-Coronagraphic-Mission/
https://en.m.wikipedia.org/wiki/Terrestrial_Planet_Finder

What about Roman? It comes with coronographs
https://www.prophy.science/article/56478000-Numerical-modeling-of-the-proposed-WFIRST-AFTA-coronagraphs-and-their-predicted-performances/
https://www.prophy.science/article/171932034-Nancy-Grace-Roman-Space-Telescope-Coronagraph-Instrument-Overview-and-Status/
https://www.prophy.science/article/172208844-End-to-end-numerical-modeling-of-the-Roman-Space-Telescope-coronagraph/
It is often quoted to be able to detect 2500 exoplanets, but a recent simulation of its bulge time domain surveys shows it can detect between 60000 to 200000 planets!! 
https://arxiv.org/abs/2305.16204
Most would be giants. Cf figure 17, it would yield ~a dozen of planets close to earth (albeit not identical), note that for Roman, only the microlensing method is able to reach planets that small.
https://www.prophy.science/article/169162029/
There is a possibility of increasing the accuracy by 5 times if the field is presighted by Euclid! 
https://arxiv.org/abs/2306.10210
Thank god it has been cited by the Euclid collaboration so this might gains traction.
Note about lsst.
https://www.prophy.science/article/169363887/
If this is true and if this scales then it follows that Xuntian could detect over 5 millions exoplanets!!! Even more since it will be the only uvoir telescope to be able to be serviced hence to have a long lifespan.
More about surveys 
https://www.prophy.science/article/179801376/
About microlensing
https://www.prophy.science/article/180411109


Note that there is also CHES which reminds me of TOLIMAN, it is a ground based? GAIA competitor that aims to reach astrometric accuracy down to 1uas which is 7 times better than GAIA end of mission forecasted accuracy.
The mission seems to be limited to a narrow field of view and for nearby planets but this is one of the most ambitious habitable planets detection missions it seems. Most importantly, with such astrometric accuracy, if they use it right, it might be the most stringent test of dark matter and general relativity.
https://www.prophy.science/article/177797375/
The intro of the companion paper is an excellent recap about the limitations of future radial velocities surveys (stellar variability) and mention other upcoming astrometric surveys, such as JASMINE (Gaia but in NIR) 
https://arxiv.org/abs/2307.05666v2
Theia 
https://arxiv.org/abs/2207.12540
Companion Paper:
https://arxiv.org/abs/2404.11210
Among those, Theia is not funded, Ches probably is but launch date unclear (can't translate the Chinese PDF...) however JASMINE while funding doesn't seems guaranteed, seems likely and the launch date is 2028.

thankfully, there is also the TIANLIN mission that will be a space based 6meter UVOIR telescope dedicated to exoplanet imaging that aims to image earth sized planet, which is currently impossible (except maybe via microlensing) 
https://arxiv.org/abs/2307.12013
Sadly this project will be launched in 10-15 years which is ineptly late for something we could do years ago but it still is significantly sooner than Habex/LUVOIR. 
Btw this very little known telescope would also happen to be the best telescope for cosmology in the 2030s

Apparently the transit method of Plato might allow over 500 earth sized planets détections, of which ~25 earth sized planets with an orbit close go earth.
While there will be many more super earth discovered with an earth like orbit, they ability to host life is limited. I have more hope for planets smaller than earth, sadly our instruments are almost unable to detect planets below earth mass, we are indeed quite lucky our instruments can target earth masses as it is.
https://arxiv.org/abs/2307.12163

Promising methods / reading list 
https://www.prophy.science/article/173210632-High-Spectral-Resolution-Dark-Holes-Concept-Results-and-Promise/
https://www.prophy.science/article/1222608-Combining-high-dispersion-spectroscopy-HDS-with-high-contrast-imaging-HCI-Probing-rocky-planets-around-our-nearest-neighbors/
https://www.prophy.science/article/2433982-Atmospheric-characterization-of-Proxima-b-by-coupling-the-Sphere-high-contrast-imager-to-the-Espresso-spectrograph/
https://www.prophy.science/article/6781801-Demonstration-of-an-efficient-photonic-based-astronomical-spectrograph-on-an-8-m-telescope/
https://www.prophy.science/article/15955383-A-laboratory-demonstration-of-the-capability-to-image-an-Earth-like-extrasolar-planet/
https://www.prophy.science/article/18044396-Large-format-platinum-silicide-microwave-kinetic-inductance-detectors-for-optical-to-near-IR-astronomy/
https://www.prophy.science/article/28635597-Medium-resolution-integral-field-spectroscopy-for-high-contrast-exoplanet-imaging-Molecule-maps-of-the-beta-Pictoris-system-with-SINFONI/
https://www.prophy.science/article/43693442-Illusion-and-reality-in-the-atmospheres-of-exoplanets/
https://www.prophy.science/article/50571655-Extreme-Adaptive-Optics/
https://www.prophy.science/article/63146310-Comparison-of-ELTs-interferometers-and-hypertelescopes-for-deep-field-imaging-and-coronagraphy/
https://www.prophy.science/article/72461381-Improving-Earth-like-planets-detection-with-an-ELT-the-differential-radial-velocity-experiment/
https://www.prophy.science/article/48373748-A-chemical-survey-of-exoplanets-with-ARIEL/
Self coherent cameras when?
https://www.prophy.science/article/91742577-The-Single-mode-Complex-Amplitude-Refinement-SCAR-coronagraph/
https://www.prophy.science/article/93132317-Minimization-of-non-common-path-aberrations-at-the-Palomar-telescope-using-a-self-coherent-camera/
Photonic
https://www.prophy.science/article/106993578-Status-and-future-developments-of-integrated-photonic-spectrographs-for-astronomy-and-Earth-and-planetary-sciences/
Benchmark https://www.prophy.science/article/118856010-Exoplanet-Imaging-Data-Challenge-benchmarking-the-various-image-processing-methods-for-exoplanet-detection/
https://www.prophy.science/article/138546587-The-vector-apodizing-phase-plate-coronagraph-design-current-performance-and-future-development/

https://www.prophy.science/article/141938973-The-ExoLife-Finder-ELF-Telescope-new-adaptive-optics-and-hybrid-dynamic-live-optical-surfaces-strategies/
https://www.prophy.science/article/142671500-Millisecond-Exoplanet-Imaging-I-Method-and-Simulation-Results/
https://www.prophy.science/article/142759730-Lunar-or-space-based-hypertelescope-for-direct-high-resolution-imaging/

Future optimization
https://www.prophy.science/article/144952019-Theoretical-performance-limits-for-coronagraphs-on-obstructed-and-unobstructed-apertures-how-much-can-current-designs-be-improved/
https://www.prophy.science/article/158324264-SHARK-VIS-ready-for-the-stars-instrument-description-and-final-laboratory-performance-test/
https://www.prophy.science/article/159287192-MagAO-X-current-status-and-plans-for-Phase-II/
https://www.prophy.science/article/159308941-Reference-star-differential-imaging-on-SPHERE-IRDIS/
https://www.prophy.science/article/159836949-SHARK-NIR-ready-to-swim-in-the-LBT-northern-hemisphere-ocean/
https://www.prophy.science/article/159872745-The-JWST-Early-Release-Science-Program-for-Direct-Observations-of-Exoplanetary-Systems-I-High-Contrast-Imaging-of-the-Exoplanet-HIP-65426-b-from-2-16-mu-m/
> These contrast limits provide sensitivity to sub-Jupiter companions with masses as low as 0.3MJup beyond separations of ∼100 au

Despite not having a dedicated high cadence and high contrast camera, JWST is able to and has directly imaged planets, it can image planets 100 times the mass of earth which is quite impressive, meaning earth sized direct imaging is realistically achievable with a lower budget than JWST.
https://www.prophy.science/article/171782054-Vortex-Fiber-Nulling-for-Exoplanet-Observations-Implementation-and-First-Light/
Optical MKIDs??
https://www.prophy.science/article/173756326-Development-of-MKIDs-in-the-Optical-and-Near-Infrared-Bands-for-SPIAKID/
Evidence for a primordial alignment in planets
https://www.prophy.science/article/177863345
https://www.prophy.science/article/168138159
New technology for spectrographs
https://x.com/terrahunting/status/1773764693489844278

https://arxiv.org/abs/2208.02720

Advances in microlensing
https://arxiv.org/abs/2304.04605
https://arxiv.org/abs/2407.06689

Occurrences rates
https://exoplanetarchive.ipac.caltech.edu/docs/occurrence_rate_papers.html

We don't have much data yet on low mass planets but it is a misconception that earth like planets are rare, we have simply no way to know and are instrument bottlenecked but based on the frequency mass evolution observed on larger planets, it seems to imply earth like planets should be "frequent"!
If this is true then the real bottleneck would either be the strict need for a G2V star and in addition, peculiar, rare chemical compositions of the planet and atmosphere + need for an exomoon?
Even so, it might be, which would be crazy, that habitable planets are very frequent in the milky way but that they are simply inhabited because both Abiogenesis and panspermia are highly unlikely (even though to have appeared extremely extremely soon on earth)
Exoplanets observations are though currently showing us planets are more diverse than what is found on our solar system, be it the abundance of super earths, maybe meta-moons, eye lens planets, tidal locking but most importantly, the possible abundance of earth sized planets that are gasous.
We might simply be among the first beings in the history of the universe, or at least of the Milky Way, to have the ability to expand.

Edit honorable mention for GRAVITY+
Which is the new SOTA in relative astrometry
https://arxiv.org/abs/2301.08071

**Comment by RedshiftDrift** [comment]
"Webb’s mysterious little red dots may be hiding entire galaxies"
https://www.sciencedaily.com/releases/2026/08/260815064808.htm

That's a more reasonable interpretation than *early stages of supermassive black holes*, or *supermassive population III stars*, or *black holes surrounded by a gaseous envelope*, or *early globular clusters in formation*, or *direct collapse black holes*, or *low-spin dark matter halos*, or other similar BS.

Instead, the results of this paper is closer to my interpretation of Little Red Dots: **a distant galaxy** :wink: [discussion 367; comment-16950550](https://github.com/orgs/a-cosmology-group/discussions/367#discussioncomment-16950550)

**Reply by HanDeBruijn** [reply]
Little Red Dot _imagined_ as a Black Hole Star.
Picture from [Scientias](https://scientias.nl/astronomen-hebben-een-nieuw-type-object-ontdekt-in-de-ruimte-de-zwartegatster/):

<img width="230" height="160" alt="sort-hul-stjerne-lead_2" src="https://github.com/user-attachments/assets/e8e8b9f5-de32-47bb-86af-ccdaa30efdbf" />

**Reply by RedshiftDrift** [reply]
"Een nieuwe klasse objecten?" Not at all!
Strange, [the image in "The Guardian"](https://www.theguardian.com/science/2026/aug/12/astronomy-discovery-new-cosmic-object-black-hole-star) is similar but not exactly the same.  I guess the artists copy each other (or is it AI generated?)
